Makefile

```diff
@@ -252,11 +246,10 @@ $(DEV_CREATE_TABLES_SCRIPT) $(DEV_CREATE_STANDARD_YML_SCRIPT): setup.py $(BUILD_
 	$(VENV_BIN)/python $< develop
 
 .PHONY: build
-build: install $(DEV_CREATE_TABLES_SCRIPT) $(DEV_CREATE_STANDARD_YML_SCRIPT)
+build: install $(DEV_CREATE_TABLES_SCRIPT) $(DEV_CONFIGURATION_YML) create_dev_db_scripts
```

Comment:
Shouldn't the build also depends on the target `development.ini`? I get a file not found error if I don't call it explicitly

Reply:
Thats true. Thankyou.

Reply:
Also did update this. Can you recheck?